contentgrid-appserver-domain/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     api project(':contentgrid-appserver-application-model')
     api project(':contentgrid-appserver-query-engine-api')
     api 'com.contentgrid.hateoas:contentgrid-pagination-api'
+    api 'com.fasterxml.jackson.core:jackson-databind'
```

Comment:
This is a bit of a huge dependency to add to the domain.

I understand you did this to get the `JsonNode` object, but alternatively use the `TreeNode` interface and only add jackson-core to the dependencies